README.rst

```diff
@@ -412,10 +412,8 @@ Phoenix.
 Help and Helping
 ----------------
 
-Most discussions about Phoenix happen on the wxPython-dev google group
-(a.k.a. the wxPython-dev mail list.) If you have questions or would like to
-get involved please subscribe to the group at
-https://groups.google.com/forum/#!forum/wxpython-dev and join in.
+Discussions about Phoenix developmet happen on this Github page.  Discussions
+about Python usage happen at https://discuss.wxpython.org/.
```

Comment:
developmet -> development

I would maybe rephrase this section to something like "Please report bugs using GitHub issues.  For help using wxPython, please use the discussion forums at https://dicuss.wxpython.org/" or something similar.

Reply:
OK, I updated that.  You might consider enabling Github Discussions too.